Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -246,6 +258,30 @@ final class ProjectEditorMapper: ProjectEditorMapping {
             )
         }()
 
+        let packagesTarget: Target? = try {
+            guard let packageManifestPath else { return nil }
+            let packageVersion = try swiftPackageManagerController.getToolsVersion(at: packageManifestPath.parentDirectory)
+            let xcodeVersion = try XcodeController.shared.selectedVersion()
+            return editorHelperTarget(
+                name: "Packages",
+                filesGroup: manifestsFilesGroup,
+                targetSettings: Settings(
+                    base: [
+                        "OTHER_SWIFT_FLAGS": .array([
+                            "-package-description-version",
+                            packageVersion.description,
+                        ]),
+                        "SWIFT_INCLUDE_PATHS": .array([
+                            "/Applications/Xcode-\(xcodeVersion).app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm/ManifestAPI",
+                        ]),
```

Comment:
Amazing findings!

Reply:
Isn't this assuming Xcode install path location? For example, I usually just have my latest Xcode installed, and the path is just `/Applications/Xcode.app`.
Wouldn't be better to use the output of `xcode-select --print-path`?

Reply:
You're right 🤦‍♂️ Yes, I will update this.